chapters/lang/expressions.tex

```diff
@@ -55,23 +55,24 @@ \section{Literal expressions}
 
 A \emph{literal expression}\index{expression!literal expression} consists of one of the literal tokens. It directly describes a number or string.
 
-Literals in Intentio are always immutable, it is not possible to change value of the literal in-place.
+Literals in Intentio are always immutable, it is not possible to change the value of the literal in-place.
 
 \section{Block expressions}
 
 \begin{bnf}
   \gd{blockexpr} \eq \gp{braceblock} \\
   \\
   \gd{braceblock} \eq \term{\{} \ \gtry{\gp{blockbody}} \ \term{\}} \\
-  \gd{parenblock} \eq \term{(} \ \gp{blockbody} \ \term{)} \\
+  \gd{parenblock} \eq \term{(} \ \gtry{\gp{blockbody}} \ \term{)} \\
```

Comment:
this change allows

```rust
if () {}
while () {}
```

to be syntactically correct

Reply:
Because it is correct

Reply:
what are the semantics of this?